app/src/main/java/com/onboarding/wheatherapp/utils/StringUtils.kt

```diff
@@ -0,0 +1,22 @@
+package com.onboarding.wheatherapp.utils
+
+object StringUtils {
+
+    private const val START_HOUR = 5
+    private const val START_DATE = 0
+    private const val EMPTY_STRING = ""
+
+    fun String.getForecastDate(): String{
```

Comment:
as this is an extension function, should be outside the StringUtils object.

Reply:
Solved!